### PR TITLE
Changes slug

### PR DESCRIPTION
**Pull request by bdamerac**
Addresses #238

### CODE REVIEW

**Comment by evykassirer** [comment]
k I'm confused

"WARNING: Unable to find `/content/S2016/S16-updated-event-wics-csc.md`, skipping url replacement"

@fboxwala @ehashman  ?

File: content/S2016/S16-updated-event-wics-csc.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ title: WiCS & CSC Go Outside
 Date: 2016-07-08
 Category: Events
 Tags: social
-Slug: wics-csc-s16
+Slug: new-wics-csc-s16
```
Comment (annalorimer) [inline content/S2016/S16-updated-event-wics-csc.md]: Change "new" to "updated" to match the event name?

**Comment by annalorimer** [comment]
Is there supposed to be no content in the file (other than the bullet points and header)?

**Comment by evykassirer** [comment]
you can extend the file to see the rest of the file content

**Comment by fboxwala** [comment]
hotfixed: c703dfd

**Pull request closed without merging**